java-squid/src/main/java/org/sonar/java/se/SymbolicValue.java

```diff
@@ -108,29 +131,19 @@ public ProgramState setConstraint(ProgramState programState, BooleanConstraint b
         if (programState.values.containsValue(this)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1066)